tests/testsuite/doc.rs

```diff
@@ -1722,14 +1787,32 @@ fn doc_private_ws() {
         .file("b/src/bin/b-cli.rs", "fn main() {}")
         .build();
     p.cargo("doc --workspace --bins --lib --document-private-items -v")
-        .with_stderr_contains(
-            "[RUNNING] `rustdoc [..] a/src/lib.rs [..]--document-private-items[..]",
-        )
-        .with_stderr_contains(
-            "[RUNNING] `rustdoc [..] b/src/lib.rs [..]--document-private-items[..]",
-        )
-        .with_stderr_contains(
-            "[RUNNING] `rustdoc [..] b/src/bin/b-cli.rs [..]--document-private-items[..]",
+        .with_stderr_data(
+            str![[r#"
+[LOCKING] 2 packages to latest compatible versions
+[DOCUMENTING] b v0.0.1 ([ROOT]/foo/b)
+[CHECKING] b v0.0.1 ([ROOT]/foo/b)
+[DOCUMENTING] a v0.0.1 ([ROOT]/foo/a)
+[RUNNING] `rustdoc [..] a/src/lib.rs [..]--document-private-items[..]
+[RUNNING] `rustc [..]
+[WARNING] function `p2` is never used
+ --> b/src/lib.rs:1:4
+  |
+1 | fn p2() {}
+  |    ^^
+  |
+  = [NOTE] `#[warn(dead_code)]` on by default
```

Comment:
We should use `...` instead of checking rustc output.

```suggestion
...
```

Reply:
Thanks, it has been modified.